src/Elliptic/Executables/Poisson/SolvePoisson.hpp

```diff
@@ -182,6 +161,10 @@ struct Metavariables {
       : tt::ConformsTo<Options::protocols::FactoryCreation> {
     using factory_classes = tmpl::map<
         tmpl::pair<DomainCreator<volume_dim>, domain_creators<volume_dim>>,
+        tmpl::pair<elliptic::analytic_data::Background,
```

Comment:
I made this consistent with the other two executables 👍 It's needed anyway for simplifications we have planned.

Reply:
FYI, the reason no `AnalyticSolution` was needed in the factory classes before is, that it was assumed _all_ background classes are analytic solutions. Now I switched from `InitializeAnalyticSolution` to `InitializeOptionalAnalyticSolution`, because I'm going to delete the other one in the future to simplify things.

Reply:
Okay, great!